generate-entity.js

```diff
@@ -0,0 +1,187 @@
+const inquirer = require('inquirer');
+const emojis = require('node-emoji');
+const fs = require('fs');
+const path = require('path');
+const YAML = require('json2yaml');
+
+console.log(`Hi Guy! ${emojis.random().emoji}, This is processer to generate entity for you!${
+  emojis.random().emoji
+}
+\nPlease answers the questions below to do it!${emojis.random().emoji}
+\n\n--------------------------------------------\n\n`);
+const DEFAULT_FILE_NAME = 'aloxide';
+const DEFAULT_PATH = path.resolve(__dirname);
+const ENTITY_NAME_QUESTION = "What's your entity name?\n";
+const PROP_NAME_QUESTION = "What's your property name?\n";
+const PROP_TYPE_QUESTION = "What's your property type?\n";
+const CONFIRM_PROP_QUESTION = "Do you want to continue create new Entity's property?\n";
```

Comment:
question ask for `y/n`, but when I type anythings else, it still process as case `n`